src/librustdoc/doctest.rs

```diff
@@ -1071,7 +1120,7 @@ fn doctest_run_fn(
         no_run: scraped_test.no_run(&rustdoc_options),
         merged_test_code: None,
     };
-    let res =
+    let (_, res) =
```

Comment:
Is there a reason we're not also tracking this, so we can report `(time compiling standalone tests, time compiling merged tests)`?

I believe doctests are run with some level of parallelism, so currently we would be reporting the total absolute wall clock time for the total compilation time, but for merged doctests, we're reporting the sum of the amount of time each thread spent compiling.  This inconsistency could over-report the impact of merged doctests.

Also, if parallelism is involved, then I think perhaps a more useful metric for minimizing compilation time would be how long the *longest* test took.

Reply:
We only compile and run standalone tests in parallel, merged doctests are never compiled in parallel (but still run in parallel).

We report raw numbers. Also we cannot get which tests took the longest to run as only `libtest` has this information in theory (but they don't keep track of it). Finally, I don't see how this is an issue to say how much time was spent in compiling merged doctests.